ci/tasks/light-aws/test-drivers.sh

```
@@ -0,0 +1,63 @@
+REPO_PARENT="$( cd "${REPO_ROOT}/.." && pwd )"
+
+if [[ -n "${DEBUG:-}" ]]; then
+  set -x
+  export BOSH_LOG_LEVEL=debug
+  export BOSH_LOG_PATH="${BOSH_LOG_PATH:-${REPO_PARENT}/bosh-debug.log}"
+fi
+
+tmp_dir="$(mktemp -d /tmp/stemcell_builder.XXXXXXX)"
+trap '{ rm -rf ${tmp_dir}; }' EXIT
+
+: ${aws_account_id:?must be set}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)